src/schema/slack/functions/_scripts/src/write_function_files_test.ts

```diff
@@ -0,0 +1,6 @@
+import { assertExists } from "../../../../../dev_deps.ts";
```

Comment:
Should we include all the generation script code in our code coverage?
The end user does not directly use this code

Reply:
If we have the tests written already, then sure! It seems like codecov reports uncovered lines in the generation script files, so if we decide to remove, then we should make sure to turn that off, too.

Reply:
Unit tests for the generation scripts are currently minimal, I will therefore exclude these files from our code coverage

implementing stable unit tests for this section is complicated since the returned values of the functions are strings of partially formatted typescript


In a follow up PR I've been able to implement an integration test that ensure that the generated code is valid typescript